assets/stylesheets/pages/_mkdocs.scss

```
@@ -0,0 +1,15 @@
+%mkdocs {
```

Can you convert this to a regular class?

```suggestion
._mkdocs {
```